datadog_checks_dev/datadog_checks/dev/tooling/specs/docs/consumers/readme.py

```diff
@@ -118,6 +120,9 @@ def render(self):
 
                     writer.write('\n')
 
+                    if 'sections' in section:
+                        sections.extendleft(section['sections'][::-1])
```

Comment:
we extend left backwards because it appends each element of the list from left to right. eg `sections.extendleft([s2.1, s2.2])` updates section to `[s2.2, s2.1, s3]` so we need to reverse it for correctness

Reply:
This would be good to include as a brief comment in the code.